etl/src/energuide/extractor.py

```diff
@@ -55,7 +53,12 @@ def _read_csv(filepath: str) -> typing.Iterator[reader.InputData]:
     except OverflowError:
         csv.field_size_limit(_WINDOWS_LONG_SIZE)
 
+    total_lines = 0
     with open(filepath, 'r', encoding='utf-8', newline='') as file:
+        for _ in file:
+            total_lines += 1
```

Comment:
Could replace this with `total_lines = sum(1 for _ in file)`

Reply:
I was trying to find a better way. This is 👍